lib/classes/ConfigSchemaHandler/index.js

```diff
@@ -136,6 +136,15 @@ class ConfigSchemaHandler {
             this.serverless.cli.log(`  ${message}`, 'Serverless', { color: 'orange' });
           }
         }
+        if (this.serverless.configurationInput.configValidationMode === undefined) {
+          this.serverless.cli.log(' ');
+          this.serverless._logDeprecation(
+            'CONFIG_VALIDATION_MODE_DEFAULT',
+            'Starting from next major Serverless will throw by default. Adapt to this' +
```

Comment:
Let's be more verbose what exactly will throw by default - maybe use similar phrasing as in deprecation doc, so something along those lines:
"Starting with next major, Serverless will throw on configuration errors by default..."